torch/_inductor/lowering.py

```diff
@@ -596,7 +596,7 @@ def register_pointwise(
     use_libdevice_for_f64=False,
 ):
     """A pointwise function that maps ops.{name} to inputs"""
-    name = name or aten_fn.__name__
+    name = name or aten_fn.__name__.removeprefix('special_')
```

Comment:
I would prefer passing `name` explicitly rather than having special cases for special in this function.

Reply:
Understood. Notice that there are more than 50 special functions. How about introducing `register_pointwise_special` helper function that removes the prefix with the assert that the prefix `special_` must exists. What do you think?

Reply:
Or have:
```python
register_pointwise_numeric(aten.special_modified_bessel_i0, name='modified_bessel_i0')
```

Reply:
Having `register_pointwise_numeric` accept other keyword args SGTM.